breathecode/events/views.py

```diff
@@ -131,6 +131,10 @@ def get_events(request):
     elif online_event == "false":
         lookup["online_event"] = False
 
+    if "technology" in request.GET:
+        value = request.GET.get("technology")
+        lookup["event_type__technologies__icontains"] = value
```

Comment:
It seems like you're trying to do an insensitive exact search, you should use `iexact`